scripts/jsonrpcproxy.py

```diff
@@ -111,6 +129,9 @@ def __init__(self, ipc_path):
     def is_connected(self):
         return True
 
+    def check_connection(self, timeout):
+        pass
```

Comment:
Perhaps add a todo comment here?

Reply:
I'm not sure it should be on any todo list. Looks like for named pipes on Windows it is always "connected".